app/services/payload/parser.rb

```diff
@@ -0,0 +1,18 @@
+module Payload
+  class Parser
+    class << self
+      def pull_request_ocurrence_time(payload)
```

Comment:
the 3 methods receive `payload` as a param. That looks to me like `payload` could be part of the initialization of the service